doc/modules/multiclass.rst

```diff
@@ -421,19 +421,19 @@ has no information about the other labels while the last model in the chain
 has features indicating the presence of all of the other labels. In general
 one does not know the optimal ordering of the models in the chain so
 typically many randomly ordered chains are fit and their predictions are
-averaged together.
+averaged together. (See Also :ref:`Regressor Chain <regressorchain>`)
```

Comment:
as I mentioned, I wanted this in the docstring, i.e. in the python file. numpydoc has a "see also" section.

Reply:
Sorry if I wasn't clear.